1.7/overview/high-availability.md

```diff
@@ -32,7 +32,7 @@ Fault domain isolation is an important part of building HA systems. In order to
  * Physical domains: this includes machine, rack, datacenter, region, availability zone,   and so on.
  * Network domains: machines within the same network may be subject  to network partitions. For example, a shared network switch may fail or have  invalid configuration.
 
-With DC/OS, it's recommended that masters be distributed across racks for HA, but not across DCs or regions. Agents may be distributed across regions/DCs, and it's recommended that you tag agents with attributes to describe their location. Synchronous services like ZooKeeper should also remain within the same region/DC to reduce network latency.
+With DC/OS, you can distribute masters across racks for HA, or across DCs and regions. Agents may be distributed across regions or DCs, and it's recommended that you tag agents with attributes to describe their location. Synchronous services like ZooKeeper should also remain within the same region/DC to reduce network latency. For more information, see the multi-AZ [documentation](/docs/1.7/administration/installing/high-availability-setup/).
```

Comment:
I assume we're considering DCs and regions equivalent, so I think we should avoid saying we recommend running masters across DCs/regions.

Can we clarify on what 'DC' means with respect to zone/regions?